electrum/util.py

```diff
@@ -681,7 +682,14 @@ def format_satoshis(
     integer_part, fract_part = result.split(".")
     if len(fract_part) < num_zeros:
         fract_part += "0" * (num_zeros - len(fract_part))
-    result = integer_part + DECIMAL_POINT + fract_part
+    # add whitespaces as thousands' separator for better readability of numbers
+    if add_thousands_sep:
+        sign = integer_part[0]
```

Comment:
what if `is_diff==False` and `x>=0`? then this will be the first digit, not a plus sign.
```
>>> from electrum import util
>>> util.format_satoshis(5, add_thousands_sep=True)
'00.000 000 05'
```